pkg/lib/serverTime.js

```diff
@@ -374,12 +374,7 @@ export function ServerTimeConfig() {
     let ntp_status = null;
     if (ntp && ntp.active) {
         let icon; let header; let body = ""; let footer = null;
-        if (ntp.synch) {
-            icon = <InfoCircleIcon className="ct-info-circle" />;
-            header = _("Synchronized");
-            if (ntp.server)
-                body = <div>{cockpit.format(_("Synchronized with $0"), ntp.server)}</div>;
```

Comment:
Hmm, I kinda like that warm fuzzy feeling about "yes, your clock is okay". Does it get in the way somehow?

Reply:
It doesn't get in the way, but it does feel like it's signalling something while everything is ok.